rustler_mix/lib/mix/tasks/rustler.new.ex

```diff
@@ -24,7 +24,6 @@ defmodule Mix.Tasks.Rustler.New do
 
   for {format, source, _} <- @basic do
     unless format == :keep do
-      @external_resource Path.join(root, source)
```

Comment:
This and the other occurrences of `@external_resource` will be readded.

Reply:
Hmm, actually I don't like this, it would mean (if I understand things correctly) that automatic recompilation is only triggered if `lib.rs` is changed, not if any of its dependencies (like other `.rs` files or a crate dependencies) are changed, is that right?

Reply:
Yes, I'll do this via `__mix_recompile__?` instead.

Reply:
`__mix_recompile__?` seems very new, I think we might not be able to support older versions of Elixir with that anymore?

Reply:
I just found it in the sources. Worst case: It won't be better than what we have now :)

The issue is that you can't really tell `mix` up-front what exact files will influence the final object files of a Rust project. I'll think about it.